templates/dns.yml.erb

```diff
@@ -5,7 +5,7 @@
 #   <%= "dns_" %>dnscmd (Microsoft Windows native implementation)
```

Comment:
leftover of f4d1c44, which could have been deleted with 8346937, I'll remove it here, thanks! :)